src/main/java/org/cloudfoundry/promregator/cfaccessor/ReactiveCFAccessorImpl.java

```diff
@@ -95,6 +96,15 @@
 	@Value("${promregator.internal.preCheckAPIVersion:true}")
 	private boolean performPrecheckOfAPIVersion;
 	
+	@Value("${cf.request.rateLimit:0}") 
+	private double requestRateLimit;
+	
+	@Value("${cf.request.backoff.initial:500}") 
+	private long firstBackoffMillis;
+	
+	@Value("${cf.request.backoff.max:1200}")
+	private long maxBackoffMillis;
```

Comment:
I absolutely want to move to duration everywhere in `simplifiedProm`

Reply:
Why do you think that they'd migrate them? As far as I understood, not specifying any unit implies "milliseconds" - see also https://www.baeldung.com/configuration-properties-in-spring-boot#property-conversion

Reply:
I didn't now about the `no unit implies millis`. I learned something new!

For the migration I was referring to the name mainly since we wouldn't want to put `Millis` in the property name if it was a duration.

Reply:
> I didn't now about the no unit implies millis. I learned something new!
 
great! 😄 

> For the migration I was referring to the name mainly since we wouldn't want to put Millis in the property name if it was a duration.

That anyhow is only internal. The external names `cf.request.backoff.initial` and `cf.request.backoff.max` (consciously) do not contain "millis".
But I am fine with it to change that internal name. 
Adjusted also with 94b52c193e7ed359f88a2f234238b8a02a65d557

Reply:
> That anyhow is only internal. The external names `cf.request.backoff.initial` and `cf.request.backoff.max`

Oh yeah. I had forgotten that. :D  Sounds good!